src/frontend/templates/header.html

```diff
@@ -42,7 +42,7 @@
                 {{ if $.show_currency }}
                 <div class="h-controls">
                     <div class="h-control">
-                        <img src="/static/icons/Hipster_CurrencyIcon.svg" alt="" class="icon" />
+                        <span class="currencyLogo"> {{ renderCurrencyLogo $.user_currency}}</span>
```

Comment:
Is there a style guide or linting for these HTML templates, out of curiosity?
```suggestion
                        <span class="currencyLogo">{{ renderCurrencyLogo $.user_currency }}</span>
```

Reply:
Not that I know of, I typically just reference the godoc https://golang.org/pkg/html/template/